resources/custom.Makefile

```diff
@@ -0,0 +1,91 @@
+# This includes all the preprocessing not directly related to the PhEval Running
+
+configurations/semsim1/%.owl:
```

Comment:
Where is the semsim1 config in the overall pipeline config yaml file?

Reply:
This is what I've mentioned to you...
Every target in this file appears to require filling with Yaml parameters, and no target will stay in this custom file.